src/pages/accountSetup/AccountSetupPage.jsx

```diff
@@ -0,0 +1,378 @@
+                          aria-describedby="headers-font-helper-text"
+                          id="headers-font-helper-text"
+                          formik={formik}
+                          fieldName="headingsFont"
+                          helperText={
+                            <StyledFormHelperText id="headers-font-helper-text">
+                              {formik.touched.headingsFont &&
+                                formik.errors.headingsFont}
+                            </StyledFormHelperText>
+                          }
+                        />
+                        <StyledFormHelperText id="headers-font-helper-text">
+                          {formik.touched.headingsFont &&
+                            formik.errors.headingsFont}
+                        </StyledFormHelperText>
+                      </FormControl>
+                    </Grid>
+
+                    <Grid item xs={12} sm={6}>
+                      <Typography variant="subtitle1" sx={{ fontSize: 20 }}>
+                        Set heading point size
+                      </Typography>
+                    </Grid>
+                    <Grid item xs={12} sm={6}>
+                      <FormControl
+                        fullWidth
+                        error={
+                          formik.touched.headingPointSize &&
+                          Boolean(formik.errors.headingPointSize)
+                        }
+                      >
+                        <CustomSelect
+                          options={pointSizeOptions}
+                          aria-describedby="headers-font-helper-text"
+                          id="headers-font-helper-text"
+                          formik={formik}
+                          fieldName="headingPointSize"
+                        />
+                      </FormControl>
+                      <StyledFormHelperText id="heading-point-size-helper-text">
+                        {formik.touched.headingPointSize &&
+                          formik.errors.headingPointSize}
+                      </StyledFormHelperText>
+                    </Grid>
+                    <Grid item xs={12} sm={6}>
+                      <Typography variant="subtitle1" sx={{ fontSize: 20 }}>
+                        Select the main text font
+                      </Typography>
+                    </Grid>
+                    <Grid item xs={12} sm={6}>
+                      <CustomSelect
+                        options={mainTextOptions}
+                        aria-describedby="main-text-font-text"
+                        id="main-text-helper-text"
+                        formik={formik}
+                        fieldName="mainTextFont"
+                        helperText={
+                          <StyledFormHelperText id="main-text-font-helper-text">
+                            {formik.touched.mainTextFont &&
+                              formik.errors.mainTextFont}
```

Comment:
The errors for this do not seem to work for some reason Select the main text font. I wonder if `formik.touched.mainTextFont` has not been set given if you try and submit the form with these empty no error ever appears. The form is just submittable with feels a little odd